src/app/type/ListItem.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
```

Comment:
'use client' may not be needed here.

Reply:
Updated: https://github.com/tsoibet/kaado/pull/7/commits/22eb888a2db1d2594e87eea92642811ca86efdae